### PR TITLE
Feature/admin panel

### PR DESCRIPTION
**Pull request by Junaid-ahmed58**

### CODE REVIEW

File: app/controllers/admin/cars_controller.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+class Admin::CarsController < AdminController
+  def index
+    @cars = Car.all
+  end
+
+  def show
+    @car = Car.find(params[:id])
```
Comment (burraqxprolabs) [inline app/controllers/admin/cars_controller.rb]: Please shit this into call back

File: app/controllers/admin/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+class Admin::UsersController < AdminController
+  def index
+    @users = User.all
+  end
+  def show
```
Comment (burraqxprolabs) [inline app/controllers/admin/users_controller.rb:8]: use this syantax

def show: end

File: app/views/shared/_admin_footer.erb [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+          </div>
+        </div>
+      </div>
+    </section>
+    <div class="text-center p-4" style="background-color: rgba(0, 0, 0, 0.025);">
+      © 2023 Copyright:
+      <a class="text-reset fw-bold" href="http://localhost:3000/cars">Pakwheels_clone.com</a>
+    </div>
+  </footer>
+</div>
```
Comment (burraqxprolabs) [inline app/views/shared/_admin_footer.erb]: Add line end here

File: config/routes.rb [hunk 4]
```diff
@@ -1,5 +1,5 @@
 Rails.application.routes.draw do
-  devise_for :users
+  devise_for :users, controllers: { sessions: 'users/sessions' }
```
Comment (burraqxprolabs) [inline config/routes.rb:2]: Why we use this controllers in routes?

Comment (Junaid-ahmed58) [reply config/routes.rb:2]: We want some changing in login page of user. To use that only controller we can add it in routes.

File: app/controllers/admin/cars_controller.rb [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+  before_action :set_admin_car, only: [:destroy , :show]
+
+  def index
+    @cars = Car.all
+  end
+
+  def show; end
+
+  def destroy
+    @car = Car.find(params[:id])
```
Comment (burraqxprolabs) [inline app/controllers/admin/cars_controller.rb]: When we move this line in before_action so why you write here again?

**Pull request merged**